packages/web-react/src/components/Card/stories/Card.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import { Markdown } from '@storybook/blocks';
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import { Direction } from '../../../constants';
+import { Button } from '../../Button';
+import { Container } from '../../Container';
+import { PartnerLogo } from '../../PartnerLogo';
+import Card from '../Card';
+import CardBody from '../CardBody';
+import CardEyebrow from '../CardEyebrow';
+import CardFooter from '../CardFooter';
+import CardLogo from '../CardLogo';
+import CardMedia from '../CardMedia';
+import CardTitle from '../CardTitle';
+import { LOGO, MEDIA_IMAGE } from '../demo/constants';
+import ReadMe from '../README.md';
+
+const meta: Meta<typeof Card> = {
+  title: 'Components/Card',
+  component: Card,
+  parameters: {
+    docs: {
+      page: () => <Markdown>{ReadMe}</Markdown>,
+    },
+  },
+  argTypes: {
+    direction: {
+      control: 'select',
+      description: 'Direction of the card.',
+      options: [...Object.values(Direction), 'horizontal-reversed'],
+      table: {
+        defaultValue: { summary: Direction.VERTICAL },
+      },
+    },
+    elementType: {
+      control: 'text',
+    },
+    isBoxed: {
+      control: 'boolean',
+      description: 'Border around the card.',
+      table: {
+        defaultValue: { summary: 'false' },
+      },
+    },
+  },
+  args: {
+    direction: 'vertical',
+    elementType: 'article',
+    isBoxed: false,
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof Card>;
+
+export const CardComponent: Story = {
```

Comment:
```suggestion
export const Playground: Story = {
```

so the URL is the same as elsewhere